frontend/src/routes/index.js

```diff
@@ -1,12 +1,24 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { BrowserRouter, Route, Redirect } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
 
-import PrivateRoute from 'components/PrivateRoute';
 import Login from 'views/Login';
 import RegisterUser from 'views/RegisterUser';
+import { getUsernameFromCookie } from 'utils/generalUtils';
+import { useAxiosTwentyMin } from 'services/customAxios';
+import PrivateRoute from 'components/PrivateRoute';
 import AppRoutes from './AppRoutes';
 
 function Routes() {
+  const dispatch = useDispatch();
+  const userData = getUsernameFromCookie();
+  const requestUserConf = { url: `users/${userData}` };
+  const [{ data }] = useAxiosTwentyMin(requestUserConf);
+
+  useEffect(() => {
+    if (data) dispatch({ type: 'SET_USER', payload: data });
```

Comment:
I think here we can do
```suggestion
    data && dispatch({ type: 'SET_USER', payload: data });
```